misc/config/config.estargz.yaml

```diff
@@ -26,7 +26,11 @@ provider:
         auth_header: header
     localhost:
       auth: YWRtaW46SGFyYm9yMTIzNDU=
+  # work directory of acceld
   work_dir: /tmp
+  gcpolicy:
+      # maximum capacity of the local caches storage
```

Comment:
size threshold that triggers GC, the oldest used blobs will be reclaimed if exceeds the size.